samples/cellular/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -367,6 +368,179 @@ To monitor the GNSS module (for instance, to see whether A-GPS or P-GPS assistan
+
+.. important::
+   The :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` Kconfig option is not secure, and should be used only for demonstration purposes!
+   This setting stores your device's private key in unprotected program memory, making your device vulnerable to impersonation.
+
+If you are certain you understand the inherent security risks, you can provision your device with :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` as follows:
+
+First, download and install the `nRF Cloud Utilities <nRF Cloud Utilities documentation_>`_ repo.
+
+Next, use the nRF Cloud Utilities :file:`create_ca_cert.py` Python script to generate a self-managed CA certificate:
+
+.. code-block:: console
+
+   python3 create_ca_cert.py -c US -f self_ca_
+
+Remember to set ``-c`` to your two-letter country code.
+See `Create CA Cert <nRF Cloud Utilities Create CA Cert_>`_ for more details.
+
+You should now have three files:
+
+  * :file:`self_ca_<certificate_serial>_ca.pem`
+  * :file:`self_ca_<certificate_serial>_prv.pem`
+  * :file:`self_ca_<certificate_serial>_pub.pem`
+
+Where ``<certificate_serial>`` is your certificate's serial number in hex.
+You will use these files to sign device certificates.
+
+You only need to generate these three files once.
+They can be used to sign as many device certificates as you need.
+
+Next create a (globally unique) `device ID <nRF Cloud Device ID_>`_ for each device you wish to provision.
+The ID can be anything you want, as long as it is not prefixed with ``nrf-`` and is globally unique.
+Each device should have its own device ID, and you must use this device ID exactly in all of the remaining steps for that device, otherwise provisioning will fail.
+
+Next, use the :file:`create_device_credentials.py` Python script to generate a self-signed certificate for each device you wish to provision:
+
+.. code-block:: console
+
+   python3 create_device_credentials.py -ca "self_ca_<certificate_serial>_ca.pem" -ca_key "self_ca_<certificate_serial>_prv.pem" -c US -cn "<device_uuid>" -f cred_
+
+Where ``<device_uuid>`` is the device ID you created, and :file:`self_ca_<certificate_serial>_ca.pem`, and :file:`self_ca_<certificate_serial>_prv.pem` are the corresponding file names from the previous step.
+Remember to set ``-c`` to your two-letter country code.
+See `Create Device Credentials <nRF Cloud Utilities Create Device Credentials_>`_ for more details.
+
+If this succeeds, you should now have an additional three files:
+
+  * :file:`cred_<device_uuid>_crt.pem`
+  * :file:`cred_<device_uuid>_pub.pem`
+  * :file:`cred_<device_uuid>_prv.pem`
+
+These three files can only be used by a single device.
+
+.. important::
+
+   If an attacker obtains the private key contained in :file:`cred_<device_uuid>_prv.pem`, they will be able to impersonate your device.
+   Since using hard-coded certificates does not protect this private key from extraction, this option should not be used in production!
```

Comment:
```suggestion
Next, complete the following steps for each device you wish to provision:

1. Create a globally unique `device ID <nRF Cloud Device ID_>`_ for the device.
   The ID can be anything you want, as long as it is not prefixed with ``nrf-`` and is globally unique.

   Each device should have its own device ID, and you must use this device ID exactly in all of the remaining steps for that device, otherwise provisioning will fail.

#. Use the :file:`create_device_credentials.py` Python script to generate a self-signed certificate for the device:

   .. code-block:: console

      python3 create_device_credentials.py -ca "self_ca_<certificate_serial>_ca.pem" -ca_key "self_ca_<certificate_serial>_prv.pem" -c US -cn "<device_uuid>" -f cred_

   Where ``<device_uuid>`` is the device ID you created, and :file:`self_ca_<certificate_serial>_ca.pem`, and :file:`self_ca_<certificate_serial>_prv.pem` are the corresponding file names from the previous step.
   Remember to set ``-c`` to your two-letter country code.
   See the `Create Device Credentials <nRF Cloud Utilities Create Device Credentials_>`_ section in the nRF Cloud Utilities documentation for more details.

   You should now have an additional three files:

   * :file:`cred_<device_uuid>_crt.pem`
   * :file:`cred_<device_uuid>_pub.pem`
   * :file:`cred_<device_uuid>_prv.pem`

   These three files can only be used by a single device.

   .. important::
      If an attacker obtains the private key contained in :file:`cred_<device_uuid>_prv.pem`, they will be able to impersonate your device.
      Since using hard-coded certificates does not protect this private key from extraction, this option should not be used in production!
```